src/widgets/FlexView.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/forbid-prop-types */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import { View } from 'react-native';
+import PropTypes from 'prop-types';
+
+/**
+ * Simple layout component. For use when needing a simple View with flex.
+ *
+ * @param {React.node} children       Children of this component
+ * @param {Number}     flex           The flex amount i.e. 1
+ * @param {Object}     style          An additional styles object.
+ */
+export const FlexView = ({ children, flex, style }) => {
+  const internalStyle = React.useMemo(() => ({ flex, ...style }));
```

Comment:
I can't think of a case where the style would change dynamically - if it could, then this would be needed?

```suggestion
  const internalStyle = React.useMemo(() => ({ flex, ...style }), [style]);
```

Reply:
oo nice - but actually it's worse and won't do anything.. `useMemo` iterates over the deps list, and checks each elements diff. But since there is no list, there is always a diff. whoops #1821